public/app/features/logs/components/panel/analytics.ts

```diff
@@ -0,0 +1,10 @@
+import { reportInteraction } from '@grafana/runtime';
+
+export const reportInteractionOnce = (interactionName: string, properties?: Record<string, unknown>) => {
+  const key = `logs.log-list-context.events.${interactionName}`;
```

Comment:
this `log-list-context` key might be too specific for how generic this method is

Reply:
Yeah, that was unexpected. Could have just used the interaction name, but let's keep the prefix.